src/components/shared/Entity/Details/Overview/NotificationSettings/index.tsx

```diff
@@ -0,0 +1,120 @@
+import { Box, Stack, useMediaQuery, useTheme } from '@mui/material';
+import CardWrapper from 'components/admin/Billing/CardWrapper';
+import MessageWithButton from 'components/content/MessageWithButton';
+import AlertBox from 'components/shared/AlertBox';
+import DataProcessingSetting from 'components/shared/Entity/Details/Overview/NotificationSettings/DataProcessingSetting';
+import Error from 'components/shared/Error';
+import { ErrorDetails } from 'components/shared/Error/types';
+import useInitializeTaskNotification from 'hooks/notifications/useInitializeTaskNotification';
+import { useEffect, useMemo, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+interface Props {
+    taskName: string;
+}
+
+function NotificationSettings({ taskName }: Props) {
+    const theme = useTheme();
+    const aboveMd = useMediaQuery(theme.breakpoints.up('md'));
+
+    const { createSubscription, getNotificationSubscription } =
+        useInitializeTaskNotification(taskName);
+
+    const [subscriptionExists, setSubscriptionExists] = useState<
+        boolean | null
+    >(null);
+    const [subscriptionError, setSubscriptionError] =
+        useState<ErrorDetails>(null);
+
+    const [updateSettingsError, setUpdateSettingsError] =
+        useState<ErrorDetails>(null);
+
+    useEffect(() => {
+        getNotificationSubscription().then(
+            (response) => {
+                if (!response.data) {
```

Comment:
This `if` condition and the `else if` condition below can be combined. The two scenarios required unique handling in a previous iteration, but that is no longer the case.